test/hash.spec.ts

```diff
@@ -1,39 +1,85 @@
 import * as assert from 'assert'
-import { keccak, keccak256, sha256, ripemd160, rlphash } from '../src'
+import {
+  keccak,
+  keccak256,
+  keccakFromString,
+  keccakFromArray,
+  sha256,
+  sha256FromString,
+  sha256FromArray,
+  ripemd160,
+  ripemd160FromString,
+  ripemd160FromArray,
+  rlphash,
+  toBuffer,
+} from '../src'
```

Comment:
Likely not worth mentioning but just in case this gets forgotten: if you add the hex string methods there should be additional test cases for the new methods.

Reply:
dbda438531f6e0d861db30a3efb1ea17b43efa8e